main/res/values/strings.xml

```diff
@@ -379,6 +379,7 @@
     <string name="list_dialog_create_err">c:geo failed to create the new list</string>
     <string name="list_dialog_remove_title">Remove list</string>
     <string name="list_dialog_remove_description">Do you want to remove the current list? All caches remaining in the list will be moved to \"Stored\".</string>
+    <string name="list_dialog_remove_nowempty">All caches in the list have been removed. Do you want to remove the current list?</string>
```

Comment:
> All caches in the list have been removed.

The result is: "List is empty." IMHO it's better to describe the current situation instead of all the events that happened to get this situation.